### PR TITLE
Ingress blog

### PR DESCRIPTION
**Pull request by pvizeli**
**Description:**

Blogpost: Hass.io Ingress

### CODE REVIEW

File: source/_posts/2019-04-15-hassio-ingress.markdown [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+comments: true
+categories: Announcements
+og_image: /images/blog/2019-04-hassio-ingress/blogpost.png
+---
+
+Today we are proud to introduce a new feature for Hass.io called Ingress. Ingress allows Hass.io add-ons to seamlessly integrate their user interface with Home Assistant. Home Assistant will take care of the authentication and the secure connection, so users can start using the add-on directly, without any configuration necessary by the users. It just works. Even with Nabu Casa’s Home Assistant Cloud Remote UI!
+
+<p class='img'>
+<img src='/images/blog/2019-04-hassio-ingress/ingress-demo.gif' alt='Ingress demo on Hass.io'>
+Demo with an Add-on on Ingress.
```
Comment (balloob) [inline source/_posts/2019-04-15-hassio-ingress.markdown]:
```suggestion
Demo with an add-on on Ingress.
```

File: source/_posts/2019-04-15-hassio-ingress.markdown [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+* [Node-RED](https://github.com/hassio-addons/addon-node-red#readme)
+* [Visual Studio Code](https://github.com/hassio-addons/addon-vscode#readme)
+* [InfluxDB](https://github.com/hassio-addons/addon-influxdb#readme)
+* [SSH & Web Terminal](https://github.com/hassio-addons/addon-ssh#readme)
+* [AdGuard Home](https://github.com/hassio-addons/addon-adguard-home#readme)
+
+You can recognize add-ons that support the Ingress feature by the Ingress icon on the add-on information tab:
+
+<p class='img'>
+<img src='/images/blog/2019-04-hassio-ingress/ingress-sign.png' alt='Add-on view with Ingress support'>
```
Comment (balloob) [inline source/_posts/2019-04-15-hassio-ingress.markdown]:
```suggestion
<img src='/images/blog/2019-04-hassio-ingress/ingress_sign.png' alt='Add-on view with Ingress support'>
```

File: source/_posts/2019-04-15-hassio-ingress.markdown [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+If an add-on is going to support both, you should not have the add-on exposed on a port enabled by default. Instead, allow users to enable the port access by assigning a port number in the “Network” section of the add-on configuration panel.
+
+Are you an add-on developer looking to support Ingress on your add-on? [Check the developer documentation][dev-docs].
+
+## {% linkable_title FAQ & Known issues %}
+
+Hass.io Ingress is a new and complex technology. Without a doubt, now that everybody starts using it, we will discover new issues. Here are some frequently asked questions and some currently known issues with the Ingress feature.
+
+* **After upgrading the add-on, I’m unable to access it directly.**</br>
```
Comment (balloob) [inline source/_posts/2019-04-15-hassio-ingress.markdown]:
```suggestion
* **After upgrading the add-on, I’m unable to access it directly.**<br>
```

File: source/_posts/2019-04-15-hassio-ingress.markdown [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+Direct access to the add-on might now be disabled by default by the add-on developer. If the add-on supports running Ingress + direct access, you can re-enable direct access by setting a port number in the “Network” section of the add-on configuration.
+
+* **After upgrading the add-on, my `panel_iframe` doesn’t work anymore.**</br>
```
Comment (balloob) [inline source/_posts/2019-04-15-hassio-ingress.markdown]:
```suggestion
* **After upgrading the add-on, my `panel_iframe` doesn’t work anymore.**<br>
```

File: source/_posts/2019-04-15-hassio-ingress.markdown [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+This is related to the question above. Once you enable direct access again, your iFrame panel will start working again. Until the automated panel integration lands in 0.92, you can also manually add a panel that points at Ingress.
+
+* **I cannot upgrade my add-on: “This update is no longer compatible with your system.”**</br>
```
Comment (balloob) [inline source/_posts/2019-04-15-hassio-ingress.markdown]:
```suggestion
* **I cannot upgrade my add-on: “This update is no longer compatible with your system.”**<br>
```

File: source/_posts/2019-04-15-hassio-ingress.markdown [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+Please update your Home Assistant installation to 0.91.3 or higher.
+
+* **I cannot access the add-on via Ingress using the Tor Browser or Firefox.**</br>
```
Comment (balloob) [inline source/_posts/2019-04-15-hassio-ingress.markdown]:
```suggestion
* **I cannot access the add-on via Ingress using the Tor Browser or Firefox.**<br>
```

File: source/_posts/2019-04-15-hassio-ingress.markdown [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+
+## {% linkable_title For add-on developers %}
+
+Ingress is added as an additional feature that add-on authors can choose to support starting today, granted that the application in the add-on supports it. The old approach of exposing add-ons on different ports will remain available. It will be up to add-on authors to choose what to support, including an option to support both.
+
+If an add-on is going to support both, you should not have the add-on exposed on a port enabled by default. Instead, allow users to enable the port access by assigning a port number in the “Network” section of the add-on configuration panel.
+
+Are you an add-on developer looking to support Ingress on your add-on? [Check the developer documentation][dev-docs].
+
+## {% linkable_title FAQ & Known issues %}
```
Comment (balloob) [inline source/_posts/2019-04-15-hassio-ingress.markdown]:
```suggestion
## {% linkable_title FAQ & Known Issues %}
```

File: source/_posts/2019-04-15-hassio-ingress.markdown [hunk 8]
```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: "Introducing Hass.io Ingress"
+description: "Ingress allows Hass.io add-ons to seamlessly integrate their user interface with Home Assistant"
+date: 2019-04-15 00:00:00
+date_formatted: "April 15, 2019"
+author: Pascal Vizeli
+author_twitter: pvizeli
+comments: true
+categories: Announcements
```
Comment (balloob) [inline source/_posts/2019-04-15-hassio-ingress.markdown:10]: Announcements or should this be Release Notes?

Comment (pvizeli) [reply source/_posts/2019-04-15-hassio-ingress.markdown:10]: I think `Announcements` because it's not direct bounded to a single release?

**Review by frenck — APPROVED** [review]

**Pull request merged**